Robust.Client/Graphics/Clyde/GLObjects/Clyde.ShaderProgram.cs

```diff
@@ -349,6 +349,33 @@ private void SetUniformDirect(int slot, in Color color, bool convertToLinear=tru
                 }
             }
 
+            public void SetUniform(string uniformName, in Color[] colors)
```

Comment:
This should not be passed via `in`.
```suggestion
            public void SetUniform(string uniformName, Color[] colors)
```